.github/workflows/ccpp.yml

```diff
@@ -266,7 +266,7 @@ jobs:
               exit 0; \
             fi
 
-  build-and-check-gcc-48:
+  build-and-check-gcc-54:
```

Comment:
I thought that `gcc-48` already was a bit unclear; can we just make that `gcc-5` now?

Do we need to accordingly update `build-and-check-gcc-48` in `.github/bors.toml`?

Reply:
Nice catch on bors!

Reply:
Ah, missed this comment. No sure how I'm managed this, sorry. I think bors.yaml can go away.